### PR TITLE
Watch page layout (desktop)

### PR DESCRIPTION
**Pull request by matedyl**
https://app.clickup.com/t/2zh2dhz

### CODE REVIEW

**Review by Esser50K — COMMENTED** [review]
still have to checkout the branch and take a look

**Review by Esser50K — CHANGES_REQUESTED** [review]
Tested it now.

I see that the video recommendations are on the right side now, but it won't collapse to a single column. I'd prefer to get this right in this PR because I'd like that every commit on Master is deployable (even when I don't deploy it)

File: app/app/src/components/AsciiPlayer.tsx [hunk 1]
```diff
@@ -137,21 +137,22 @@ function AsciiPlayer(props: AsciiPlayerProps) {
...
+        let newFontSize: number
 
         // this is the character ratio of a base monospace font
         const characterRatio = 5 / 3
         const windowPortion = 0.75
         const ratio = lineLength / nLines
-        const heightLimited = (lineLength / window.innerWidth) < (nLines / window.innerHeight)
-        const limiter = heightLimited ? window.innerHeight : window.innerWidth
+        const heightLimited = (lineLength / playerContainer.current!.clientWidth) < (nLines / window.innerHeight)
+        const fontAdjustment = 1.1  // scale adjustment for fitting the text block to the container width - depends on the width of the character in font family
```
[the remark this answers is not in the document]
Comment (matedyl) [reply app/app/src/components/AsciiPlayer.tsx:148]: @Esser50K I added it to fit the YT layout as closely as possible and due to the way font rendering works: it is not fully responsive and renders font in the same size for a range of `font-size` values. It has some implications on how closely the design can match YT watch page. Let's schedule a call soon and I can explain it further!

**Comment by matedyl** [comment]
> 



> Tested it now.
> 
> I see that the video recommendations are on the right side now, but it won't collapse to a single column. I'd prefer to get this right in this PR because I'd like that every commit on Master is deployable (even when I don't deploy it)

Could you explain what you mean by collapsing to a single column? After testing I see that the main layout is a column: with video on the top and recommended vids section on the bottom.

**Review by Esser50K — APPROVED** [review]

**Pull request merged**